c_glib/gandiva-glib/node.cpp

```diff
@@ -1529,3 +1685,35 @@ ggandiva_if_node_new_raw(std::shared_ptr<gandiva::Node> *gandiva_node,
                               NULL);
   return GGANDIVA_IF_NODE(if_node);
 }
+
+GGandivaAndNode *
+ggandiva_and_node_new_raw(std::shared_ptr<gandiva::Node> *gandiva_node,
```

Comment:
How about just defining `ggandiva_boolean_node_new_raw()` instead of defining `ggandiva_{and,or}_node_new_raw()`?

Reply:
I've defined `ggandiva_boolean_node_new_raw()`.